specs/darwin/crash_logs.table

```diff
@@ -0,0 +1,27 @@
+table_name("crash_logs")
+description("Application Crash Logs")
+schema([
+    Column("process", TEXT, "Name of the process that crashed."),
```

Comment:
Yeah, I think it might be useful. We expose pid in a lot of other tables and, although you would never have a process in this table and the `processes` table at the same time, perhaps you could match them in some event based table retroactively or what have you.